Mathlib/Algebra/Polynomial/RingDivision.lean

```diff
@@ -750,6 +750,13 @@ theorem comp_eq_zero_iff : p.comp q = 0 ↔ p = 0 ∨ p.eval (q.coeff 0) = 0 ∧
       Or.rec (fun h => by rw [h, zero_comp]) (fun h => by rw [h.2, comp_C, h.1, C_0]) h
 #align polynomial.comp_eq_zero_iff Polynomial.comp_eq_zero_iff
 
+lemma aeval_ne_zero_of_isCoprime [CommSemiring R] [Nontrivial S] [Semiring S] [Algebra R S]
```

Comment:
What about a name that allows dot notation?

Reply:
Do you think `IsCoprime.aeval_ne_zero` is better than `Polynomial.aeval_ne_zero_of_isCoprime`?

Reply:
Mmm, you're right, we also want `Polynomial`. Let's keep as it is then.